integration-tests/package.json

```diff
@@ -13,6 +13,7 @@
     "build:contracts": "hardhat compile",
     "build:contracts:ovm": "hardhat compile --network optimism",
     "test:integration": "hardhat --network optimism test",
+    "test:sync": "hardhat --network optimism test sync-tests/*.spec.ts --no-compile",
```

Comment:
I feel like we need sync tests that have a dummy db that is passed to geth (upstream geth in L1 mode because hardhat doesn't allow for this) and then we sync from that deterministically. Perhaps we can call these verifier tests? We need to do the exact same logic for replicas so maybe network tests? Since its a network of nodes?

Reply:
^can rename to network tests!

Reply:
I don't think we necessarily need a pre-instantiated db as things are reasonably deterministic in our current envs? Think PR is fine rn otherwise

Reply:
That was more of a longer term comment, sorry for the confusion. In the long term we need sync tests that basically sync thousands of blocks where we can observe things like performance and state

Reply:
^opted to rename those later since sync is clearer for this context :)